tests/cypress/e2e/unit_tests/special_fleet_tests.spec.ts

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
Check unnecessary spaces in the file and remove them.

Reply:
Can you pls be more specific where?